setup.cfg

```diff
@@ -2,18 +2,11 @@
 name = pyirf
 description = Python IACT IRF builder,
 url = https://github.com/cta-observatory/pyirf
-author = Julien Lefaucheur, Michele Peresano, Thomas Vuillaume
+author = Julien Lefaucheur, Michele Peresano, Thomas Vuillaume, Maximilian Nöthe
```

Comment:
with this massive refactoring, I think you can remove the author list and use the contributors one from GitHub...

Reply:
This is in setup.cfg for the pypi metadata, not the docs. It should contain the names of the 
current maintainers, or  maybe even something like "CTA Consortium"

Reply:
I changed it to cta aswg